lib/WP/Functions.php

```diff
@@ -470,4 +470,16 @@ function getPostTypes() {
   function getTerms() {
     return call_user_func_array('get_terms', func_get_args());
   }
+
+  function getEditableRoles() {
+    return call_user_func_array('get_editable_roles', func_get_args());
+  }
+
+  function getCategories() {
+    return call_user_func_array('get_categories', func_get_args());
+  }
+
+  function hasFilter() {
+    return call_user_func_array('has_filter', func_get_args());
+  }
```

Comment:
This is a huge commit that I'm unable to review completely and I believe you've use some automatization to replace all of this. That's why it would be nice next time if the commit was really atomic and did only what it says in the commit message. Here is some other code appended but if there is any other code in other places I'm unable to discover that.

Reply:
I was unable to do that manually so I did it via a script, but the script messed up some formatting (add/remove empty lines). And I was also unable to do it in multiple commits. I am sorry for that.

Reply:
It's OK to have the result of the script in one single commit, I was just pointing out these methods above seem to be added manually so it seems to me kind of mixed-in.